src/edu/csus/ecs/pc2/core/imports/clics/CLICSProblem.java

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 1989-2022 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.imports.clics;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.model.JSONObjectMapper;
+
+/**
+ * A JSON representation for the CLICS API problems json data.
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ *
+ */
+public class CLICSProblem {
+
+    // {time_limit=1, color=null, name=Match Game, id=matchgame, label=A, test_data_count=48, rgb=null, ordinal=1}
+    @JsonProperty
+    private String id;
+
+    @JsonProperty
+    private String name;
+
+    @JsonProperty
+    private String time_limit;
+
+    @JsonProperty
+    private String color;
+
+    @JsonProperty
+    private String label;
+
+    @JsonProperty
+    private Integer test_data_count;
+
+    @JsonProperty
+    private String rgb;
+
+    @JsonProperty
+    private Integer ordinal;
+
```

Comment:
I think the missing fields from the CLICS 2202-07 spec (`uuid`, `max_score`, `package`, and `statement` ) should be included in this class.

Reply:
Out of scope, added above notes into #559

Reply:
Again, I disagree this is out of scope.  The tool is supposed to read the CLICS event feed; the CLICS event feed could contain those fields.  What happens in that case?

Reply:
This PR/change is already 2,500 lines and 23 classes - the missing work is covered by #559.